src/chain.js

```diff
@@ -137,23 +137,7 @@ process.on("message", function(message) {
   }
 });
 
-
-function sendHeartBeat() {
-  process.send({type: 'heartbeat'})
-}
-
-sendHeartBeat()
-
-// Keep an interval going so the process stays open.
-// We can use this interval to maintain a heartbeat
-setInterval(function() {
-  sendHeartBeat()
-}, 1000);
-
-process.on('uncaughtException', (err) => {
-  console.log(err.stack || err)
-  process.send({type: 'error', data: err.stack || err})
-});
```

Comment:
This blcok is really important for any other uncaught errors. Without this, failures we haven't accounted for won't get bubbled up to the user (for instance, see comments below). We should keep this block.

Reply:
Doh! Good catch, man. I can't believe I deleted both blocks.